data/data/powervs/loadbalancer/alb.tf

```diff
@@ -118,3 +118,31 @@ resource "ibm_is_lb_pool_member" "api_member" {
   target_address = local.api_servers[count.index]
 }
 
+# bootstrap listener and backend pool
```

Comment:
Not sure how cleanup things works for the bootstrap but we need to ensure we clean these resources and remove 22 from the security group also post bootstrap completion.

Reply:
good catch. i created a work item for this